### PR TITLE
Append challenge link to the description #72

### PR DESCRIPTION
**Pull request by aquariuscd**

### CODE REVIEW

File: bot/cogs/cyber.py [hunk 1]
```diff
@@ -90,6 +90,7 @@ def __init__(self, bot: Bot):
             challenge_title = challenge_raw["title"]
             challenge_difficulty = challenge_raw["difficulty"]
             challenge_text = challenge_raw["description"]
+            challenge_text += f"\nhttps://assess.joincyberdiscovery.com/challenge-{challenge_num:02d}"
```
Comment (AnonGuy) [inline bot/cogs/cyber.py]: Looks good to me. This is a pretty nice use of f-strings.

Comment (picapi) [inline bot/cogs/cyber.py]: Great idea! Could it be more user friendly to utilise the URL property of embeds, making them easily clickable, without the need for a full link URL to be visible in the description text area?

https://discordpy.readthedocs.io/en/rewrite/api.html#discord.Embed.url

Comment (joker314) [reply bot/cogs/cyber.py]: Alternatively, embeds support better markdown

Comment (joker314) [reply bot/cogs/cyber.py]: Alternatively, embeds support better markdown

Comment (joker314) [reply bot/cogs/cyber.py]: Alternatively, embeds support better markdown

**Pull request merged**